maint_tools/test_docstrings.py

```diff
@@ -88,7 +88,6 @@
     "LabelSpreading",
     "Lars",
     "LarsCV",
-    "Lasso",
```

Comment:
It seems that you are modifying RFE but you remove Lasso. Could you fix it.

Reply:
I made some mistakes on managing the branch of my previous pull request.